frontend/src/scss/main.scss

```diff
@@ -56,4 +56,17 @@ svg path {
 
 .v-btn {
     text-decoration: none !important;
-}
\ No newline at end of file
+}
+
+.content-with-padding {
+    padding: 70px 66px 0 80px;
```

Comment:
what are the magic numbers?) And why is bottom without padding?

Reply:
There is really small distance from header to content.

Reply:
And these styles are probably better

width: 90%;
    margin: 0 auto;
    padding: 32px 0;